Proiect1/Proiect1/CircusModel/Circus.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Proiect1.Animals;
+using Proiect1.Interface;
+using Proiect1.Presentation;
+using System.Collections.Generic;
+
+namespace Proiect1.CircusModel
+{
+    internal class Circus
```

Comment:
nobody said anything about a circus; this is not in the requirements

Reply:
I changed that, now it's definitely not a circus